snaps/concepts/keyring-api.md

```diff
@@ -7,78 +7,65 @@ sidebar_custom_props:
 
 # About the Keyring API
 
-:::flaskOnly
-:::
-
-:::tip API documentation
-See the [Keyring API reference](../reference/keyring-api/index.md) for all the Keyring API methods.
-:::
-
-The Snaps Keyring API integrates custom EVM accounts inside MetaMask.
-Previously, you needed a companion dapp to display custom EVM accounts, such multi-party computation
-(MPC) accounts.
-Now you can display these custom accounts alongside regular MetaMask accounts in the UI:
+The Keyring API integrates custom EVM accounts inside MetaMask.
+You can use the Keyring API to display custom accounts, such as multi-party computation (MPC)
+accounts, alongside regular MetaMask accounts in the user interface:
 
 <p align="center">
-<img src={require('../assets/keyring/accounts-ui.png').default} alt="Keyring Snap accounts in Metamask UI" width="360" />
+<img src={require('../assets/keyring/accounts-ui.png').default} alt="Account management Snap accounts in Metamask UI" width="360" style={{border: '1px solid gray'}} />
 </p>
 
-[Create a Keyring Snap to integrate custom EVM accounts in MetaMask.](../tutorials/custom-evm-accounts.md)
-Your dapp can then use the [`eth_requestAccounts`](/wallet/reference/eth_requestaccounts) MetaMask
-JSON-RPC API method to connect to the custom accounts, and seamlessly interact with them using other
-[JSON-RPC methods](/wallet/reference/json-rpc-api).
+To use the Keyring API, you first [implement the API in an account management Snap](../how-to/use-keyring-api/snap/index.md)
+(also known as an "account Snap").
+Your dapp can then [invoke Keyring API methods](../how-to/use-keyring-api/dapp.md) on the account
```

Comment:
We've been calling the dapp that iteracts directly with the Snap a "companion dapp" to differentiate from dapps that iteract with the account through MetaMask.